From patchwork Sat Oct  7 14:02:21 2017
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: bfq: Fix bool initialization/comparison
From: Thomas Meyer <thomas@m3y3r.de>
X-Patchwork-Id: 9991351
Message-Id: <1507383097082-1668690714-1-diffsplit-thomas@m3y3r.de>
To: paolo.valente@linaro.org, axboe@kernel.dk,
 linux-block@vger.kernel.org, linux-kernel@vger.kernel.org
Date: Sat, 07 Oct 2017 16:02:21 +0200

Bool initializations should use true and false. Bool tests don't need
comparisons.

Signed-off-by: Thomas Meyer <thomas@m3y3r.de>

diff -u -p a/block/bfq-iosched.c b/block/bfq-iosched.c
--- a/block/bfq-iosched.c
+++ b/block/bfq-iosched.c
@@ -4986,7 +4986,7 @@ static ssize_t bfq_low_latency_store(str
 
 	if (__data > 1)
 		__data = 1;
-	if (__data == 0 && bfqd->low_latency != 0)
+	if (__data == 0 && bfqd->low_latency)
 		bfq_end_wr(bfqd);
 	bfqd->low_latency = __data;
 
